IPython/terminal/console/interactiveshell.py

```diff
@@ -115,6 +115,14 @@ def _client_changed(self, name, old, new):
         self.session_id = new.session.session
     session_id = Unicode()
 
+    @property
+    def own_kernel(self):
+        return self._own_kernel
+
+    @own_kernel.setter
+    def own_kernel(self, b):
+        self._own_kernel = bool(b)
```

Comment:
This property does seem redundant. The only two places you're setting it, it's a boolean already.